external/ilc-assets/ilc-assets.depproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <NugetRuntimeIdentifier>win10-x64-aot</NugetRuntimeIdentifier>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uapaot'">
+    <FileToInclude Include="System.Diagnostics.Tracing" />
```

Comment:
I'm not entirely sure what this is for. Is this the netcore50 assets? which is what it looks like in which case I think I would call this uap something as opposed to ilc. We could also put this into the harvesting depproj that Eric recently added and only bin-place these for this configuration.